webapp/documentation/test_scenarios.md

```diff
@@ -0,0 +1,24 @@
+# Navbar
+  - Signed in state
+    - [ ] Auth: Nav bar should contain user auth (profile photo, name, and sign out). Sign out should kick the user to the authentication page.
+    - [ ] User presence: when another user concurrently uses nook, the person indicator (colored dots) appears. Clicking on them should take to the respective conversation.
+        - Test by adding a conversation id and the time stamp under `user_presence > user email`
+    - [ ] Conversation list panel: A dropdown appears when there are more than 1 conversation shards present.
+	- Test by adding a conversation name in firebase under `nook_conversation_shards > shard-n > name (field), conversations (collection)`
+    - [ ] Style: Nav bar shouldn't hide any of the content below
+
+  - Signed out state
+    - [ ] Navbar should contain only sign-in button, should log in via the first domain name
```

Comment:
I think it might look better not to be a navbar at all in sign-out state, just the login page - what do you think?

Reply:
(this is what I ended up doing in the merge, but would appreciate your thoughts on it, am happy to revert the change!)

Reply:
Yep that makes sense! I just noticed it along with the comment